hs-abci-extra/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright Charles Crain (c) 2019
```

Comment:
Should we have licenses in subfolders? 
Also as I remember we use Apache

Reply:
We can remove this auto generated license since we have the one a level up

Reply:
Ahh yeah this was from `stack new`

Reply:
Ahh yeah this was from `stack new`